scripts/lease-email-config.json.template

```diff
@@ -0,0 +1,13 @@
+{
+    "smtp_server": "XXXXXXXXXXXXXXXXXXXXXXX",
+    "smtp_port": XXX,
+    "smtp_username": "XXXXXXXXXXXXXXXXXXXXXXXXX",
+    "smtp_password": "XXXXXXXXXXXXXXXXXXXXXXXXX",
+    "smtp_from": "esi@massopen.cloud",
+    "lease_filter": {
+	"filter": "expiring",
+	"within": 4
+    },
+    "subject_template": "[MOC ESI] Expiring Leases for Project {project}",
+    "body_template": "You are receiving this reminder email because you have leased nodes in the MOC ESI that will expire soon. Please contact the MOC ESI administrators if you would like to extend these leases.\n\n{lease_table}"
```

Comment:
We might have to have a footer in these emails that says something along the lines of `You are receiving this email because you are a user of the MOC ESI cluster. If you would like to stop receiving email you will need to delete your account by contacting <EMAIL>`. I'm not 100% sure if this is required but CAN-SPAM act requires something like this for commercial email, I'm just not sure if this is considered commercial. The only thing could be if we start charging for bare-metal. @msdisme any experience here?

Reply:
We will be charging at some point and might as well get ahead of it.  

Does the email server support an email remove solution (eg. reply with unsubscribe in the body of the mail)?  If so that, plus keeping a list of who chooses to be dropped, seems like the best approach.   

If this becomes too time-consuming, it may be handled in a future PR. In that case, support@massopen.cloud or nerc's email  is a good starting point.

Reply:
That's not something the server would do. For an unsubscribe functionality we'd need to host a web server and generate tokens for each email that goes out for each user with an unsubscribe link and also build that into this script. Not very straight forward. I think the regulation allows for the unsubscribe method to be an email request so we should probably just go with support@massopen.cloud for now.

Reply:
We could always keep a custom unsubscribe list as a file, and have the script check emails against that. I'd vote for making that future update.

Reply:
I've added the suggested footer to the examples